float64[3] relPosNED	# NED component of relative position vector (m)
